src/internal_events/throttle.rs

```diff
@@ -9,7 +9,15 @@ pub(crate) struct ThrottleEventDiscarded {
 
 impl InternalEvent for ThrottleEventDiscarded {
     fn emit(self) {
-        debug!(message = "Rate limit exceeded.", key = ?self.key); // Deprecated.
+        // TODO: Technically, the Component Specification states that the discarded events metric
```

Comment:
Hmm, yeah, this is a good question. I wonder how much harm there is in allowing additional tags as long as the required tags are there.